src/hott/06-contractible.rzk.md

```diff
@@ -553,3 +549,157 @@ together the two identifications to the out and back path.
+        ( transport-lift A C
+            ( x)
+            ( center-contraction A is-contr-A)
+            ( rev
+              ( A)
+              ( center-contraction A is-contr-A)
+              ( x)
+              ( homotopy-contraction A is-contr-A x))
+            ( u))))
+
+#def has-section-total-type-to-second-center-is-contr-base
+  ( A : U)
+  ( is-contr-A : is-contr A)
+  ( C : A → U)
+  : has-section
+    ( Σ ( x : A) , C x)
+    ( C (center-contraction A is-contr-A))
+    ( total-type-to-second-center-is-contr-base A is-contr-A C)
+  :=
+    ( ( second-center-to-total-type-is-contr-base A is-contr-A C)
+    , \ u →
+      ( concat
+        ( C (center-contraction A is-contr-A))
+        ( total-type-to-second-center-is-contr-base A is-contr-A C
+          ( second-center-to-total-type-is-contr-base A is-contr-A C u))
+        ( transport A C
+          ( center-contraction A is-contr-A)
+          ( center-contraction A is-contr-A)
+          ( first-path-Σ A C (center-contraction A is-contr-A , u)
+            ( center-contraction A is-contr-A , u)
+            ( refl))
+          ( u))
+        ( u)
+        ( transport2
+          ( A)
+          ( C)
+          ( center-contraction A is-contr-A)
+          ( center-contraction A is-contr-A)
+          ( rev
+            ( A)
+            ( center-contraction A is-contr-A)
+            ( center-contraction A is-contr-A)
+            ( homotopy-contraction A is-contr-A
+              ( center-contraction A is-contr-A)))
+          ( refl)
+          ( all-paths-equal-is-contr
+            ( A)
+            ( is-contr-A)
+            ( center-contraction A is-contr-A)
+            ( center-contraction A is-contr-A)
+            ( rev
+              ( A)
+              ( center-contraction A is-contr-A)
+              ( center-contraction A is-contr-A)
+              ( homotopy-contraction A is-contr-A
+                ( center-contraction A is-contr-A)))
+              ( refl))
+            ( u))
+        ( second-path-Σ A C
+          ( ( center-contraction A is-contr-A) , u)
+          ( ( center-contraction A is-contr-A) , u)
+          ( refl))))
+
+#def equiv-total-type-second-center-is-contr-base
+  ( A : U)
+  ( is-contr-A : is-contr A)
+  ( C : A → U)
+  : Equiv
+    ( Σ ( x : A) , C x)
+    ( C (center-contraction A is-contr-A))
+  :=
+    ( ( total-type-to-second-center-is-contr-base A is-contr-A C)
+    , ( ( ( has-retraction-total-type-to-second-center-is-contr-base
+            A is-contr-A C)
+      , ( has-section-total-type-to-second-center-is-contr-base
+            A is-contr-A C))))
+```
+
+Any two elements in a contractible type are equal, so we extend the equivalence
+to the dependent type evaluated at any given term in the base.
+
+```rzk
+#def transport-equiv-total-type-second-center-is-contr-base
+  ( A : U)
+  ( is-contr-A : is-contr A)
+  ( C : A → U)
+  ( x : A)
+  : Equiv
+    ( Σ ( x : A) , C x)
```

Comment:
Very minor comment, feel free to ignore: here `x` shadows outer `x`. I recommend renaming the outer `x : A` to `a : A`.

Reply:
No problem, it's done.